test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -396,8 +396,9 @@ def test_make_date_prop(self) -> None:
             "2022",
             "GREGORIAN:CE:0476-09-04:CE:0476-09-04",
             "GREGORIAN:CE:2014-01-31",
+            "JULIAN:BC:1:AD:200",
         ]
-        invalid_values = ["GREGORIAN:CE:0476-09-04:CE:09-04", "GREGORIAN:CE:0476-09-010:CE:0476-09-04"]
+        invalid_values = ["GREGORIAN:CE:0476-09-010:CE:0476-09-04"]
```

Comment:
Unfortunately, if we accept years of 1-3 digits, this no longer produces an error. Because 09 could be the year, 04 the month, and the day could be omitted... So we just have to live with the fact that if we allow 1-3 digit years, we have to make validation less strict.

Reply:
no we can specify in a regex that we do not allow numbers to start with a 0 through [1-9]

Reply:
But this won't resolve the problem at hand, or would it?